src/styles/button.css

```diff
@@ -1 +1,60 @@
 /* button */
+
+.ui.button {
+  background-color: var(--primary-pms-362_900);
+  color: var(--shade-white);
+  font-family: var(--gotham);
+  border-radius: var(--border-radius-3);
+}
+
+/** Focus **/
+.ui.button:focus {
+  background-color: var(--primary-pms-362_900);
+  color: var(--shade-white);
+}
+
+/** Hover **/
+
+.ui.button:hover {
+  background-color: var(--primary-pms-362_800);
+  color: var(--shade-white);
+}
+
+/** Active **/
+
+.ui.button:active {
+  background-color: var(--primary-pms-362_700);
+  color: var(--shade-white);
+}
+
+/** Secondary Button **/
+
+.ui.secondary.button {
+  background-color: var(--shade-white);
+  color: var(--primary-pms-362_900);
+  border: 1px solid var(--primary-pms-362_900);
+}
+
+/** Secondary Focus **/
+
+.ui.secondary.button:focus {
+  background-color: var(--shade-white);
+  color: var(--primary-pms-362_900);
+  border: 1px solid var(--primary-pms-362_900);
+}
+
+/** Secondary Hover **/
+
+.ui.secondary.button:hover {
+  background-color: var(--shade-white);
+  color: var(--primary-pms-362_800);
+  border-color: var(--primary-pms-362_800);
+}
+
+/** Secondary Active **/
+
+.ui.secondary.button:active {
+  background-color: var(--shade-white);
+  color: var(--primary-pms-362_700);
+  border-color: var(--primary-pms-362_700);
```

Comment:
Just note that these will need to be updated with the new renamed variables that use hyphens.

Reply:
Yeah! I will merge the dropdown when it's approved and then rebase!